### PR TITLE
sig-scale takes ownership of scalability tests

### PR DESCRIPTION
**Pull request by shyamjvs**
To make our tests more accountable.

@gmarek SGTY?

### CODE REVIEW

File: jobs/config.json [hunk 1]
```diff
@@ -1838,7 +1838,7 @@
     ],
     "scenario": "kubernetes_e2e",
     "sigOwners": [
-      "UNKNOWN"
+      "sig-scale"
```
Comment (gmarek) [inline jobs/config.json]: I'm not sure about taking ownership of release branch tests. Can we skip those for now?

Comment (shyamjvs) [inline jobs/config.json]: SGTM. Changed them.

**Comment by shyamjvs** [comment]
cc @kubernetes/sig-scalability-misc @countspongebob

**Pull request merged**